tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/dynamic_hero_block.html

```diff
@@ -1,17 +1,28 @@
-<div>
-    {% if value.static_text %}<span>{{ value.static_text }}</span>{% endif %}
+<div data-dynamic-hero class="dynamic-hero">
+    {% if value.static_text %}<h2 class="heading heading--three heading--light">{{ value.static_text }}</h2>{% endif %}
     {% if value.dynamic_text %}
         {% if value.dynamic_text|length > 1 %}
-            {# If there's more than one dynamic text, show the controls for the loop. #}
-            <ul>
-                {% for text in value.dynamic_text %}
-                    <li>{{ text }}</li>
-                {% endfor %}
-            </ul>
+            {# If there's more than one dynamic text, show Swiper carousel structure and controls for the loop. #}
+            <div class="swiper">
+                <ul class="swiper-wrapper">
+                    {% for text in value.dynamic_text %}
+                        <li class="swiper-slide">
+                            {# Fixed px values required to control the font size for all browsers and prevent slide overflow#}
+                            <h3 class="heading heading--mega text-[60px] lg:text-[100px]">{{ text }}</h3>
+                        </li>
+                    {% endfor %}
+                </ul>
+            </div>
+            <div class="dynamic-hero__controls" aria-label="Carousel controls">
+                <button class="dynamic-hero__control" aria-label="Previous message" data-dynamic-hero-prev>{% include "patterns/atoms/icons/icon.html" with name="chevron" classname="dynamic-hero__control--prev" %}</button>
+                <button class="dynamic-hero__control" aria-label="Pause autoplay" data-dynamic-hero-pause>{% include "patterns/atoms/icons/icon.html" with name="pause" classname="dynamic-hero__control--pause" %}</button>
+                <button class="dynamic-hero__control" aria-label="Resume autoplay" data-dynamic-hero-play>{% include "patterns/atoms/icons/icon.html" with name="play" classname="dynamic-hero__control--play" %}</button>
```

Comment:
Maybe some design feedback to pass on to Ben but I wonder if we need some visual state for the play and pause buttons so the user knows which one is active. A bit like the play and pause buttons under the hero video here: https://www.gosh.nhs.uk/

Reply:
This is a very good point.  Ben suggested alternating between play and pause buttons instead of showing them at the same time - the solution is now committed